salt/modules/boto_rds.py

```diff
@@ -197,6 +197,33 @@ def parameter_group_exists(name, tags=None, region=None, key=None, keyid=None,
         return resp
 
 
+def db_cluster_exists(name, tags=None, region=None, key=None, keyid=None,
+                           profile=None):
+    '''
+    .. versionadded:: Fluorine
+
+    Check to see if an RDS db cluster exists.
+
+    CLI example:
+
+    .. code-block:: bash
+
+        salt myminion boto_rds.db_cluster_exists mydbcluster \
+                region=us-east-1
+    '''
+    conn = _get_conn(region=region, key=key, keyid=keyid, profile=profile)
+
+    try:
+        rds = conn.describe_db_clusters(DBClusterIdentifier=name)
+        return {'exists': bool(rds), 'error': None}
+    except ClientError as e:
```

Comment:
I would recommend some sort of special handling for AWS API throttling errors - they're extremely common, and the general best practice is to wait-retry the call (a reasonable number of times) before failing the call entirely.

Something like this would be along the lines of what I mean:
```            if error.error_code == 'Throttling':
                log.debug('Throttled by AWS API, will retry in 5 seconds.')
                time.sleep(5)
                retries -= 1
                continue
            log.error('Error fetching config for ELB {0}: {1}'.format(name, error.message))
            log.error(error)
```  Check out the boto_ec2 or various other modules (grep for "Throttled") for good examples.